code/game/machinery/medical_kiosk.dm

```diff
@@ -9,31 +9,34 @@
 	density = TRUE
 	circuit = /obj/item/circuitboard/machine/medical_kiosk
 	payment_department = ACCOUNT_MED
-	var/default_price = 5  //I'm defaulting to a low price on this, but in the future I wouldn't have an issue making it more or less expensive.
-	var/scan_active = null  //Shows if the machine is being used. resets upon new viewer.
-	var/adv_scan_active = null  //Shows if the machine has upgraded functionality
+	var/default_price = 5           //I'm defaulting to a low price on this, but in the future I wouldn't have an issue making it more or less expensive.
+	var/active_price = 5            //Change by using a multitool on the board.
+	var/scan_active = null          //Shows if the machine is being used. resets upon new viewer.
+	var/upgrade_scan_active = null
+	var/adv_scan_active = null      //Shows if the machine has upgraded functionality
```

Comment:
are these boos >:(

Reply:
Good coding style is not something I'm great at, MB

Reply:
boos need to use `TRUE`/`FALSE` defines and should have default values of either (in this case they'd be `FALSE`)